include/crypto/x509.h

```diff
@@ -55,7 +55,7 @@ struct x509_sig_info_st {
 struct X509_req_info_st {
     ASN1_ENCODING enc;          /* cached encoding of signed part */
     ASN1_INTEGER *version;      /* version, defaults to v1(0) so can be NULL */
-    X509_NAME *subject;         /* certificate request DN */
+    const X509_NAME *subject;   /* certificate request DN */
```

Comment:
I don't think we want to make a change like that. This can potentially break other applications.

Reply:
This seems to be a local include file, that's fine then.